website/static/js/share/utils.js

```diff
@@ -190,30 +228,269 @@ utils.removeFilter = function(vm, filter){
     utils.search(vm);
 };
 
-utils.arrayEqual = function(a, b) {
+/* Tests array equality */
+utils.arrayEqual = function (a, b) {
     return $(a).not(b).length === 0 && $(b).not(a).length === 0;
 };
 
-utils.loadStats = function(vm){
-    vm.statsLoaded(false);
+utils.addFiltersToQuery = function (query, filters) {
+    if (filters) {
+        filters.forEach(function (filter) {
+            query = utils.filteredQuery(query, filter.filter);
```

Comment:
Sorry for not seeing this earlier guys, but this function is no longer used and cane be deleted